web-ui/modules/i18n/locales/en.mjs

```
@@ -0,0 +1,1091 @@
+    'common.refresh': 'Refresh',
+    'common.refreshing': 'Refreshing...',
+    'common.loading': 'Loading...',
+    'common.saving': 'Saving...',
+    'common.sending': 'Sending...',
+    'common.scanning': 'Scanning...',
+    'common.export': 'Export',
+    'common.import': 'Import',
+    'common.apply': 'Apply',
+    'common.applying': 'Applying...',
+    'common.confirming': 'Confirming...',
+    'common.writeToEditor': 'Write to editor',
```

Biome (2.4.15)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)